en/docs/reference/connectors/google-pubsub-inbound/google-pubsub-inbound-endpoint-example.md

```diff
@@ -0,0 +1,94 @@
+
+1. Setting up Google Cloud Pub/Sub resources
+2. Configure the Inbound Endpoint in WSO2 Micro Integrator using the Visual Studio Code extension
+3. Run and test the integration to receive real-time messages
+
+#### **Step 01: Setting up Google Cloud Pub/Sub resources**
+##### Prerequisites
+
+* Create a [Google Cloud Account](https://console.cloud.google.com/freetrial?facet_utm_source=google&facet_utm_campaign=%28organic%29&facet_utm_medium=organic&facet_url=https%3A%2F%2Fcloud.google.com%2Fpubsub%2Fdocs%2Fpublish-receive-messages-console&facet_id_list=%5B39300012%2C+39300020%2C+39300118%2C+39300196%2C+39300241%2C+39300319%2C+39300320%2C+39300326%2C+39300345%2C+39300354%2C+39300363%2C+39300374%2C+39300412%2C+39300421%2C+39300436%2C+39300472%2C+39300488%2C+39300496%2C+39300498%2C+39300569%5D) if you already don't have one.
+* Create/Select a [Google Cloud Project](https://console.cloud.google.com/projectselector2/home/dashboard).
+* Enable the [Pub/Sub API](https://console.cloud.google.com/flows/enableapi?apiid=pubsub.googleapis.com)
+
+##### Creating required resources
+* [Create a Topic](https://cloud.google.com/pubsub/docs/publish-receive-messages-console#create_a_topic)
+* [Create a Subscription](https://cloud.google.com/pubsub/docs/publish-receive-messages-console#add_a_second_subscription)
+* [Set Up Authentication](https://cloud.google.com/docs/authentication/set-up-adc-attached-service-account)
+
+#### **Step 02: Configure the Inbound Endpoint in WSO2 Micro Integrator using the Visual Studio Code extension**
+In MI, create a new project and add the Google Pub/Sub connector to the project.
+
+1. [Create a new project]({{base_path}}/develop/create-integration-project/) in WSO2 Micro Integrator (MI).
+2. In the Add Artifact interface, under Create an Integration, click Event Integration. This will open the list of event integrations available in WSO2 Micro Integrator.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/sf-pubsub-inbound/sf-pubsub-inb-2.png" title="Google Pub/Sub Inbound Endpoint Example" width="80%" alt="Event Integration"/>
+
+3. Select Google Pub/Sub Inbound Endpoint
+4. Fill the form with the values for creating the inbound endpoint
+
+    4.1. Google PubSub Basic Configuration consists of the basic configuration for the Google Inbound Endpoint
+         <img src="{{base_path}}/assets/img/integrate/connectors/google-pubsub-inbound/createEventIntegration.png" title="Google Pub/Sub Inbound Example" alt="Event Integration" width="80%" height="30%"/>
+
+    4.2. Advance Configuration consists of required configuration for Subscription Create/Update, Flow Control configuration and other Subscriber related configurations
+         <img src="{{base_path}}/assets/img/integrate/connectors/google-pubsub-inbound/advancedConfig.png" title="Google Pub/Sub Inbound Advanced Config" alt="Order Notification" width="80%" height="30%"/>
+
+    4.3. Subscription Create/Update configuration and Flow Control configuration
```

Comment:
Since this is the tutorial, better to have only the required configurations here and have the rest under the Reference doc.